package.json

```diff
@@ -8,7 +8,7 @@
     "start": "npm install && PORT=3000 NODE_PATH=$NODE_PATH:./src node --harmony bin/server.js",
     "lint": "eslint src",
     "lint:fix": "npm run lint -- --fix",
-    "test": "karma start",
+    "test": "NODE_PATH=./src:./test mocha --compilers js:babel-core/register --timeout 5000 --require ./test/setup.js --recursive src/**/**/__tests__/*.js",
```

Comment:
This seems like a lot of configuration in the command args. Any way we can pull this into a file, etc.?